### PR TITLE
Provide custom version of updateCheck.checkForUpdates and replace versionInfo.version only for the duration of retrieving update info

### LINKED ISSUE #5: Issue when creating portable version
Issue first opened against NVDA (https://github.com/nvaccess/nvda/issues/13786) but Update Channel Selector was discovered to cause the issue.

### STR
* On NVDA 2022.1, install update channel selector. The update channel is set to "Default" (I thought to have it on "Stable, rc and beta" but this was not the case actually)
* Download last alpha from NVDA (nvda_snapshot_alpha-25538,f3445949.exe)
* Start the launcher and create a portable copy.
* Launch the portable copy or check its content

### Actual result
* The portable copy crashes
* The folders "lib", "lib64" and "libArm64" are all empty

### Actual result
* The portable copy should start without any problem
* The whole portable copy should be created: the folders "lib", "lib64" and "libArm64" should contain a folder with files.

### PR DESCRIPTION
**Pull request by lukaszgo1**
Fixes #5 
Rather than replacing `versionInfo.version` when the plugin is initialized it is done only for the duration of retrieving update info. In the future it  can be considered to do the same for `versionInfo.updateVersionType`.
I've tested that:
- Checking updates on Alpha when the channel is set to stable proposes update to 2022.1
- When generating portable copies libraries are not excluded

cc @CyrilleB79

### CODE REVIEW

File: addon/globalPlugins/updateChannel.py [hunk 1]
```diff
@@ -88,19 +122,14 @@ def getAvailableUpdates(self, currentChannel):  # noqa C901
...
-				if originalChannel == "snapshot:alpha" and originalChannel != channel:
-					versionInfo.version = str(versionInfo.version_year) + "." + str(versionInfo.version_major) + "."\
-					+ str(versionInfo.version_minor)
 				self.availableUpdates[channel] = updateCheck.checkForUpdate()
+			except RuntimeError:  # Thrown by `updateCheck.checkForUpdate`
+				self.availableUpdates[channel] = -1  # An error occurred
+			else:
 				if not self.availableUpdates[channel]:
 					self.availableUpdates[channel] = 1  # Already updated
-			except Exception:
```
Comment (lukaszgo1) [inline addon/globalPlugins/updateChannel.py:100]: This is not strictly related, however I wasted a lot of time trying to understand why my change did not worked, and it turned out I made a stupid typo, yet the exception got swallowed making debugging unnecessarily difficult. Please try  to avoid broad except statements like these.

**Comment by jmdaweb** [comment]
Thanks, @lukaszgo1
